iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/ThreeWindingsTransformerToBeEstimated.java

```diff
@@ -0,0 +1,113 @@
+ * If a tap changer is not to be estimated, it should not be changed during a state estimation (i.e its tap position is only an input).
+ *
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface ThreeWindingsTransformerToBeEstimated extends Extension<ThreeWindingsTransformer> {
+
+    @Override
+    default String getName() {
+        return "threeWindingsTransformerToBeEstimated";
+    }
+
+    /**
+     * Return true if the ratio tap changer of the leg 1 of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsRatioTapChanger1();
+
+    /**
+     * Return true if the ratio tap changer of the leg 2 of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsRatioTapChanger2();
+
+    /**
+     * Return true if the ratio tap changer of the leg 3 of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsRatioTapChanger3();
+
+    /**
+     * Return true if the ratio tap changer of the leg of the given side of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsRatioTapChanger(ThreeWindingsTransformer.Side side);
+
+    /**
+     * Return true if the phase tap changer of the leg 1 of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsPhaseTapChanger1();
+
+    /**
+     * Return true if the phase tap changer of the leg 2 of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsPhaseTapChanger2();
+
+    /**
+     * Return true if the phase tap changer of the leg 3 of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsPhaseTapChanger3();
+
+
+    /**
+     * Return true if the phase tap changer of the leg of the given side of the extended three windings transformer is to be estimated during a state estimation.
+     * Else, return false.
+     */
+    boolean containsPhaseTapChanger(ThreeWindingsTransformer.Side side);
+
+    /**
+     * Specify if the ratio tap changer of the leg 1 of the extended three windings transformer is to be estimated during a state estimation.
+     */
+    ThreeWindingsTransformerToBeEstimated setRatioTapChanger1Status(boolean toBeEstimated);
+
+    /**
+     * Specify if the ratio tap changer of the leg 2 of the extended three windings transformer is to be estimated during a state estimation.
+     */
+    ThreeWindingsTransformerToBeEstimated setRatioTapChanger2Status(boolean toBeEstimated);
```

Comment:
Same remark here: we cannot have just less methods that takes the side in argument?